docs/content/content/content-structure.mdx

```diff
@@ -57,6 +57,8 @@ Headings should be:
 - descriptive
 - unique.
 
+It is common for screen reader users to navigate content by jumping from one heading to the next. You can nest heading levels to create subsections of content, but you should not skip heading levels as this will confuse some users.
```

Comment:
```suggestion
It’s common for screen reader users to understand a page’s context and hierarchy through its heading structure, and to navigate content by jumping from one heading to the next. You can nest heading levels to create subsections of content, but you cannot not skip heading levels as this will confuse some users.
```

Same content as PR 1794